fluxacademy.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<head>
+<title>Flux Academy</title>
+    <style>
+        body {
+            margin: 0;
+        }
+        .container {
+         padding: 1%;
+            box-sizing:border-box;
+            
+    }
+        .header {
+            height: 100px;
+            text-align:center;
+    }
+
+        .left-col, .right-col {
+        width: 25%;
+        float: left;
+    }
+        .center-col {
+            width:50%;
+            float:left;
+        }
+
+</style>
+</head>
+<body>
+    <img src="">
+    <!-- sticky nav how to -->
+    <nav>
+        <img src="" alt="logo">
+        <ul>
+            <li>Courses</li>
+            <li>Student Stories</li>
+            <li>About</li>
+            <li>Log In</li>
+        </ul>
+    </nav>
+    <div>
+        <h1>BECOME A FREELANCE WEB DESIGNER NO MATTER WHERE YOU LIVE</h1>
+        <p>You bring the creativity and passion. We teach you the skills you need to succeed as a freelance web designer. It all happens online, no matter where you are.</p>
+    </div>   
+    <div>
+        <img src="" alt="student"> 
+        <!--img overlaps video -->
+        <video width="" height="">
+            <source src="movie.mp4" type="video/mp4">
+        </video>
+    </div>
+    <div>
+        <h2>BUILD HIGH-VALUE SKILLS WITH BUSINESS AND WEB DESIGN COURSES ONLINE</h2>
+        <p>Thriving as a freelancer takes both craft and business strategy. Our courses help you with both. Become a skilled web designer and run a business you love with our proven processes and roadmaps</p>
+        <video width="" height="">
+            <source src="movie2.mp4" type="video/mp4">
+        </video>
+    </div>
+        <!--img overlaps video -->
+        <img src="" alt="student">
+    </div> 
+    <div>
```

Comment:
Missing a closing `div` for this.  When you forget to close divs, everything becomes a child of this `div`